src/inputs/select.scss

```diff
@@ -3,113 +3,38 @@
 @import '../style/index';
 @import '../menu/index';
 
-$iui-after-border: 5px solid;
-
 @mixin iui-select {
   @include iui-reset;
   display: flex;
-  flex-direction: column;
+  align-items: center;
+  user-select: none;
   position: relative;
-  width: 100%;
-  min-width: 0;
-
-  > .iui-select-button {
```

Comment:
From what I understand, your new styles are changed like this:
- `iui-select` becomes `iui-input-with-icon`
- `iui-select-button` becomes `iui-select`
- `iui-select-button::after` becomes `iui-end-icon`
- `iui-content` and `iui-icon` are preserved

Does that sound right?

Once you make the overlapping grid change mentioned in my other comment, I think you would need to revert some of that structure back (e.g. `iui-input-with-icon` will have no border).